tmail-backend/integration-tests/webadmin/postgres-webadmin-integration-tests/src/test/java/com/linagora/tmail/integration/postgres/PostgresUsernameChangeIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package com.linagora.tmail.integration.postgres;
+
+import static com.linagora.tmail.james.jmap.OpenSearchContactConfiguration.DEFAULT_CONFIGURATION;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Durations.ONE_HUNDRED_MILLISECONDS;
+
+import java.io.IOException;
+
+import org.apache.james.JamesServerExtension;
+import org.apache.james.backends.opensearch.ReactorOpenSearchClient;
+import org.awaitility.Awaitility;
+import org.awaitility.Durations;
+import org.awaitility.core.ConditionFactory;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.opensearch.client.opensearch._types.query_dsl.QueryBuilders;
+import org.opensearch.client.opensearch.core.SearchRequest;
+
+import com.linagora.tmail.DockerOpenSearchExtension;
+import com.linagora.tmail.integration.UsernameChangeIntegrationContract;
+
+@Disabled("TODO wait to taskManager https://github.com/linagora/james-project-private/issues/773#issuecomment-2014158867")
```

Comment:
Q: Is `MemoryTaskManager` not enough to pass this IT?

Reply:
I see Rene wrote memory task manager has an issue
https://github.com/linagora/james-project-private/issues/773#issuecomment-2014158867

Reply:
Maybe... very unsure, could be just honestly the postgres implem that has one :)

Please try to reenable that back

Reply:
I enabled it, but one test case did not pass 
task: https://github.com/linagora/tmail-backend/issues/994
https://github.com/linagora/james-project/issues/5143